doc/configuration.rst

```diff
@@ -316,6 +332,25 @@ All available configuration options for Ansible are:
 
       * ``env``: An optional list environment variables to set when creating the container. The format of each variable is ``var_name=value``.
 
+
+Ansible performance considerations
```

Comment:
@laurentganne @adidanes @stebenoist What do you think about having a dedicated page about performance considerations?
Could be later enriched with others considerations.

Reply:
Maybe there aren't enough materials for now, would a page just containing the same details as above be ok ?  It would give:

# Performance

## TOSCA Operations

As described in TOSCA :ref:`tosca_operations_implementations_section`, Yorc supports these builtin implementations for operations to execute on remote hosts :

  * Bash scripts
  * Python scripts
  * Ansible Playbooks

It is recommended to implement operations as Ansible Playbooks to get the best execution performance.

When operations are not implemented using Ansible playbooks, the following Ansible configuration settings allow to improve the performance of scripts execution on remote hosts :

  * `use_openssh`: Prefer OpenSSH over Paramiko, a Python implementation of SSH (used by default) to provision remote hosts. OpenSSH have several optimization like reusing connections that should improve preformance but may lead to issues on older systems See Ansible documentation on Remote connection information.
  * `cache_facts`: Caches Ansible facts (values fetched on remote hosts about network/hardware/OS/virtualization configuration) so that these facts are not recomputed each time a new operation is a run for a given deployment.
  * `archive_artifacts`: Archives operation bash/python scripts locally, copies this archive and unarchives it on remote hosts (requires tar to be installed on remote hosts), to avoid multiple time consuming remote copy operations of individual scripts.

Reply:
@adidanes @stebenoist WDYT?

Reply:
+1 for this specific perf paragraph